lib/services/access_list_test.go

```diff
@@ -22,20 +22,20 @@ import (
 
 	"github.com/stretchr/testify/require"
 
-	"github.com/gravitational/teleport/api/types"
-	"github.com/gravitational/teleport/api/types/header"
+	"github.com/gravitational/teleport/lib/types/accesslist"
+	"github.com/gravitational/teleport/lib/types/header"
 	"github.com/gravitational/teleport/lib/utils"
 )
 
 // TestAccessListUnmarshal verifies an access list resource can be unmarshaled.
 func TestAccessListUnmarshal(t *testing.T) {
-	expected, err := types.NewAccessList(
+	expected, err := accesslist.NewAccessList(
 		header.Metadata{
 			Name: "test-access-list",
 		},
-		types.AccessListSpec{
+		accesslist.AccessListSpec{
 			Description: "test access list",
-			Owners: []types.AccessListOwner{
+			Owners: []accesslist.AccessListOwner{
```

Comment:
Can we reduce name stuttering by removing `AccessList` from all of the things that are now disambiguated by package? For example:

```suggestion
			Owners: []accesslist.Owner{
```

and so on? 

(This comment really belongs in `lib/types/accesslist/accesslist.go`, but GitHub won't let me comment on unchanged lines in a PR... 🙄 )

Reply:
Done!